en/docs/tutorials/scenarios/scenario4-user-signup-approval-flow.md

```diff
@@ -1,4 +1,40 @@
+
+**_Time to Complete : 3 mins_**
+
+![Signup description]({{base_path}}/assets/img/tutorials/scenarios/signup-desc.png)
+
+WSO2 API Manager provides extension points to trigger workflow tasks for many operations such as Application creation, subscription creation, user signup, etc. By default, WSO2 API Manager comes with a simple approval workflow. 
+
+This demo setup already comes with self-signup and signup workflow enabled for the Qantis tenant domain. If you need to check how to configure this check [enabling self-signup]({{base_path}}/reference/customize-product/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup) and [enabling signup workflow]({{base_path}}/reference/customize-product/customizations/adding-a-user-signup-workflow) documentation. 
 
 !!! Note
-    Content to be added.
\ No newline at end of file
+    You could see **UserSignUpApprovalWorkflowExecutor** is now engaged if you login to the [https://localhost:9443/carbon](https://localhost:9443/carbon) using Qantis admin credentials (admin@qantis.com:admin) and navigate to `/_system/governance/apimgt/applicationdata/workflow-extensions.xml` location.
+
+    ![Signup config]({{base_path}}/assets/img/tutorials/scenarios/signup-config.png)
+
+
+To try this out,
+
+1. Go to the Developer Portal [https://localhost:9443/devportal/](https://localhost:9443/devportal/) and select Qantis tenant domain.
+2. Select Sign In button on the top right corner. This will show a signup page.
+3. Select **Create Account** Link.
+4. Add the desired username (with the tenant domain) and proceed to self register.
+
+    ![Signup config]({{base_path}}/assets/img/tutorials/scenarios/signup-start-pg.png)
+
+5. Fill the required fields and proceed with the registration.
+6. If you try to login using this credentials, you will be shown an unauthorized-access page.
+   
+    ![Not allowed]({{base_path}}/assets/img/tutorials/scenarios/login-forbidden.png)
+
+
+To approve the signup request, 
+
+1. Log on to the [https://localhost:9443/admin/](https://localhost:9443/admin/) Admin Portal using `admin@quantis.com` and password `admin`.
+2. Under the **Task → User Creation** you will notice a pending task. Approve it.
+
+    ![Approve]({{base_path}}/assets/img/tutorials/scenarios/approve.png)
+3. Now you will be able to log on to the Developer Portal using `raj@quantis.com` user and his password.
```

Comment:
Ok, got the idea with this. Shall we indicate that this is the user created earlier? Or in that earlier step we can say "Create an account called `raj@quantis.com`" or something like that.

Reply:
This user was in the image under the point 4 in previous section. but I added that user as u mentioned under that step